test-files/ui/features/folder_conflation_advanced_options.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Folder and conflation options control
+
+	Scenario: Opening Management Tab
+        Given I am on Hootenanny
+        And I click Get Started
+        Then I select the "sprocket" div
+
+    Scenario: Invalid folder names will not be added
+        When I click on the "Datasets" option in the "settingsSidebar"
+        And I click the "import-add-folder" button
+        And I fill "NewFolderName" input with "@#&%^"
+        And I press "big.loud" span with text "Add Folder"
+        Then I should see an invalid name warning for ".NewFolderName" input
+        Then I close the modal
+        Then I select the "sprocket" div
```

Comment:
I think the ticket means to use invalid names in this field of the conflate form:
![screenshot from 2017-07-19 12 30 12](https://user-images.githubusercontent.com/2600932/28378400-0ab151ce-6c7e-11e7-8530-0a6fcee53e6f.png)